### PR TITLE
add aggregator Last()

### PR DESCRIPTION
**Pull request by EaconTang**
add aggregator Last()

Last(): get the last one data on the timestamp

### CODE REVIEW

File: src/core/Aggregators.java [hunk 1]
```diff
@@ -34,7 +33,7 @@
     LERP,   /* Regular linear interpolation */
     ZIM,    /* Returns 0 when a data point is missing */
     MAX,    /* Returns the <type>.MaxValue when a data point is missing */
-    MIN     /* Returns the <type>.MinValue when a data point is missing */
+    MIN,    /* Returns the <type>.MinValue when a data point is missing */
```
Comment (johann8384) [inline src/core/Aggregators.java]: Is this change intentional?

Comment (johann8384) [inline src/core/Aggregators.java]: Is this change intentional?

Comment (EaconTang) [inline src/core/Aggregators.java]: No, that is unconscious change...
Sorry for my carelessness, shall I just modified it and send another pull request to the "next" branch?

Comment (manolama) [inline src/core/Aggregators.java]: Hi @EaconTang yeah if you could revert that then squash all your commits into one please, that'd be great.

Comment (EaconTang) [inline src/core/Aggregators.java]: @johann8384 @manolama Hey， I just found that "first" and "last" aggregators already exist in "next" branch, and the implementation code makes me a little confused...

Comment (EaconTang) [inline src/core/Aggregators.java]: https://github.com/OpenTSDB/opentsdb/blob/next/src/core/Aggregators.java

Seems like the codes in class "First" and class "Last"  are almost the same?

Comment (manolama) [inline src/core/Aggregators.java]: Ah right yeah these are already in there at https://github.com/OpenTSDB/opentsdb/blob/next/src/core/Aggregators.java#L666-L708 in 2.3.0. I'll close this PR then.

The code for those aggregators is correct. The "First" aggregator fetches the first value from the iterator then iterates over the others to consume the iterator but doesn't assign the output anywhere. The "Last" aggregator iterates similarly but always assigns the iteration value to the variable.

**Comment by manolama** [comment]
Duplicate of what's in https://github.com/OpenTSDB/opentsdb/blob/next/src/core/Aggregators.java#L666-L708
